src/sshd/install.sh

```diff
@@ -141,15 +112,6 @@ sudoIf()
 }
 
 EOF
-if [ "${FIX_ENVIRONMENT}" = "true" ]; then
```

Comment:
Can we remove this variable declaration at the top?

> FIX_ENVIRONMENT="${FIX_ENVIRONMENT:-"true"}"

Reply:
done